samples/msal-node-samples/custom-INetworkModule-and-network-tracing/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "sample",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "type": "module",
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@azure/msal-common": "^7.6.0",
```

Comment:
Can we add scripts to use the local version of msal packages as depencies? ([e.g.](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/samples/msal-node-samples/auth-code-cli-app/package.json))

Reply:
Why use the local version of msal packages instead of the version that was used to write this sample? There may be breaking changes in the future.

Reply:
Ah, you're about HttpClient -nevermind then